polytiles/polytiles.py

```diff
@@ -152,49 +148,48 @@ def close(self):
 
 # https://github.com/mapbox/mbutil/blob/master/mbutil/util.py
 class MBTilesWriter:
-    def __init__(self, filename, setname, overlay=False,
-                 version=1, description=None, format='png256'):
+    def __init__(self, filename, setname, overlay=False, version=1, description=None, format="png256"):
         self.format = format
         self.filename = filename
-        if not self.filename.endswith('.mbtiles'):
-            self.filename = self.filename + '.mbtiles'
-        self.con = sqlite3.connect(self.filename)
+        if not self.filename.endswith(".mbtiles"):
+            self.filename = self.filename + ".mbtiles"
+        self.con = sqlite3.connect(self.filename, isolation_level=None)
         self.cur = self.con.cursor()
-        self.cur.execute("""PRAGMA synchronous=0""")
-        self.cur.execute("""PRAGMA locking_mode=EXCLUSIVE""")
-        # self.cur.execute("""PRAGMA journal_mode=TRUNCATE""")
-        self.cur.execute("""create table if not exists tiles (zoom_level integer, tile_column integer, tile_row integer, tile_data blob);""")
-        self.cur.execute("""create table if not exists metadata (name text, value text);""")
-        self.cur.execute("""create unique index if not exists name on metadata (name);""")
-        self.cur.execute("""create unique index if not exists tile_index on tiles (zoom_level, tile_column, tile_row);""")
-        metadata = [('name', setname), ('format', format2ext(self.format)),
-                    ('type', 'overlay' if overlay else 'baselayer'), ('version', version)]
+        self.cur.execute("PRAGMA synchronous=0")
+        self.cur.execute("PRAGMA locking_mode=EXCLUSIVE")
+      # self.cur.execute('PRAGMA journal_mode=TRUNCATE')
```

Comment:
I doubt this would even compile. Comments adhere to the same indentation rules as code.